wandb/sdk/artifacts/artifact.py

```diff
@@ -1597,6 +1597,19 @@ def download(
             path_prefix=path_prefix,
         )
 
+    def change_collection_type(self, new_collection_type: str) -> None:
```

Comment:
I think this method should be on the [collection](https://github.com/wandb/wandb/blob/ef00e4f6b3be64a95a322353a874b818b9c901fa/wandb/apis/public/artifacts.py#L306) instead